pkg/gadgets/trace/exec/types/types.go

```diff
@@ -15,19 +15,32 @@
 package types
 
 import (
+	"strings"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32   `json:"pid,omitempty"`
-	Ppid      uint32   `json:"ppid,omitempty"`
-	UID       uint32   `json:"uid,omitempty"`
-	MountNsID uint64   `json:"mountnsid,omitempty"`
-	Retval    int      `json:"ret,omitempty"`
-	Comm      string   `json:"pcomm,omitempty"`
-	Args      []string `json:"args,omitempty"`
+	Pid       uint32   `json:"pid,omitempty" column:"pid,width:7,fixed"`
+	Ppid      uint32   `json:"ppid,omitempty" column:"ppid,width:7,fixed"`
+	Comm      string   `json:"pcomm,omitempty" column:"comm,width:16,fixed"`
+	Retval    int      `json:"ret,omitempty" column:"ret,width:3,fixed"`
+	Args      []string `json:"args,omitempty" column:"args,width:40"`
+	UID       uint32   `json:"uid,omitempty" column:"uid,width:10,fixed,hide"`
```

Comment:
```bash
$ tput cols
95
$ ./kubectl-gadget trace exec -n test-ns
NODE        NAMESPACE  POD        CONTAINER  PID     PPID    COMM             RET ARGS
worker      test-ns    test-…test test-pod-… 195427  155649  sleep            0   /bin/sleep 0…
worker      test-ns    test-pod   test-pod   195443  100031  sleep            0   /bin/sleep 0…
```

I think COMM is taking too much space after making it fixed. Maybe we could do something like mix=10(?) and max=16?

While for ARGS, we definitely need a minimum.

Reply:
Yes, that can be a good idea. Anyway, I think we can handle those tune ups later on when we convert all gadgets to use this library.